packages/compass-components/src/components/leafygreen.tsx

```diff
@@ -112,6 +112,29 @@ delete (Checkbox as React.ComponentType<any>).propTypes;
 // all hrefs.
 export { Link, Button, IconButton } from './links/link';
 
+// Working around leafygreen lack of support for `defaultValue` property
```

Comment:
Asked about it [here](https://mongodb.slack.com/archives/CFFA5BS79/p1745424399561789), if they open a ticket, we can link it here

Reply:
Okay, we have a ticket to follow now https://jira.mongodb.org/browse/LG-5063